### ISSUE TITLE
Unable to install SABNZBd

### ISSUE THREAD

**Issue by craigsta37** [SynoCommunity/spksrc#3545]
I am trying to install SABNZBd and am getting the following error message:

The following packages are missing in the package server. Please contact the Synology support team for help.
python

I have a Synology DS118 with the latest version of DSM. I have tried to install the Python3 package that is in the Synocommunity package centre but am still unable to install SABNZBd.

Any help with what Python is required would be most appreciated.

**Comment by BenjV** [comment]
You need Python 2 for SabNzdb, it does not run under Python 3.

**Comment by craigsta37** [comment]
> You need Python 2 from the SynoCommunity for SabNzdb, it does not run under Python 3.

Thanks BenjV. Where do i download this from?

**Comment by BenjV** [comment]
If you can find SabNzbd and install it you should also be able to find Python.

Assuming you added the SynoCommunity to your package center
(e.g. http://packages.synocommunity.com)

**Comment by craigsta37** [comment]
> If you can find SabNzbd and install it you should also be able to find Python.
> 
> Assuming you added the SynoCommunity to your package center
> (e.g. http://packages.synocommunity.com)

Yes i did add Synocommunity to my package centre. There is Python3 in the list of packages that appear in the "community" tab, but no Python 2 that i can see. Is there somewhere else where i would get this from?

**Comment by BenjV** [comment]
You are right is am also missing the python 2 package for the rtd1296 architecture at the moment.
@ymartin59 , @Safihre Could this be added to the package server?

**Comment by MasterXL6** [comment]
Sorry if this isn't the right place to ask this question.

But I've recently bought a Synology 418j and noticed that most packages that are CPU specific are missing this rtd1296 (armv8) branch. Did I make a mistake buying this unit, is it too niche/not well supported by the community? (I'm still in my return window)


Also I have to add, I setup my box on the 8th of december, and I was able to install python 2, sabnzbd and couchpotato. On monday (10th) I wanted to install hydra and the installer said it couldn't find python in the community package list. (while it was actually already installed!)
I made the mistake of deinstalling everything and now I obviously can't install anything that's python 2 dependent :(

Does this mean the rtd1296 (armv8) package was temporary removed from https://synocommunity.com/package/python?
